photon-client/src/integTest/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverIntegTest.scala

```diff
@@ -433,7 +433,7 @@ class GameTrainingDriverIntegTest extends SparkTestUtils with GameTestUtils with
 
     runDriver(args.put(GameTrainingDriver.rootOutputDirectory, outputDir))
 
-    compareModelEvaluation(new Path(outputDir, "best"), trainedMixedModelPath, 0.005)
+    compareModelEvaluation(new Path(outputDir, "best"), trainedMixedModelPath, 0.007)
```

Comment:
This value shouldn't need to be updated, unlike the others. The other values are RMSE scores which the result should either match or improve upon. This value is an epsilon value for comparing two RMSE values.

Reply:
Addressed comments